ttnn/cpp/ttnn/operations/conv/conv2d/device/optimized_conv_op_program_factory.cpp

```diff
@@ -46,6 +47,24 @@ pair<vector<uint32_t>, vector<uint32_t>> compute_opt_conv_activation_as_mm_shape
     return {{1, num_rows_padded, num_cols_padded}, {1, num_rows, num_cols}};
 }
 
+pair<vector<uint32_t>, vector<uint32_t>> compute_opt_conv_activation_as_mm_shape(const Shape& conv_activation_shape, ttnn::operations::sliding_window::SlidingWindowConfig sliding_window_config, uint32_t act_block_h_ntiles, uint32_t padding_for_32B_alignment) {
```

Comment:
is the other function with same name (with conv params) still needed?

Reply:
The old convolution still uses the version of this function with `conv_params`.